<launch>
<!-- Start Gazebo with wg world running in (max) realtime -->
<include file="$(find navigation)/launch/rolling_landscape_120m.launch"/>



<!--	<node pkg="gazebo_ros" type="gzserver" name="gazebo_server" 
	<node pkg="gazebo_ros" type="gzserver" name="gazebo_server" 
	      args="$(find pioneer3at)/config/gazebo/world2.sdf"/>
-->
<!-- Spawn simulated quadrotor uav -->
<include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
  <arg name="x" value="0.0"/>
  <arg name="y" value="0.0"/>
  <arg name="z" value="0.3"/>
  <arg name="model" value="$(find hector_quadrotor_description)/urdf/quadrotor_hokuyo_utm30lx.gazebo.xacro"/>
</include>
<!-- Start rviz visualization with preset config -->
<node pkg="rviz" type="rviz" name="rviz" args="-d $(find hector_quadrotor_demo)/rviz_cfg/outdoor_flight.rviz"/>
</launch>
